File: redis/scala/read/redis_read.scala
// ------------------------------------------------------------------    
//	redis_read.scala
//
//					Apr/11/2013
//
// ------------------------------------------------------------------    
import redis.clients.jedis.Jedis

// ------------------------------------------------------------------    
object redis_read
{

// ------------------------------------------------------------------    
def main (args: Array[String])
{
	println ("*** 開始 ***")

	var jedis = new Jedis("host_dbase")

	val set_keys = jedis.keys("*")

	val keys = set_keys.toArray ()
	for (key <- keys)
		{
		val str_key = key.toString ()
		val str_json = jedis.get (str_key)

		if (str_json != null)
			{
			redis_manipulate.record_parse_proc (str_key,str_json)
			}
		}

	println ("*** 終了 ***")        

}
// ------------------------------------------------------------------    
}
// ------------------------------------------------------------------    

File: redis/java/read/redis_read.java
// -------------------------------------------------------------
//
//	redis_read.java
//
//						Jan/12/2020
// -------------------------------------------------------------
import redis.clients.jedis.Jedis;

import	java.util.HashMap;
import	java.util.Set;
// -------------------------------------------------------------
public class redis_read {

// -------------------------------------------------------------
public static void main(String[] args)
{
	System.out.println ("*** 開始 ***");

	Jedis jedis = new Jedis("localhost");

	Set set_keys = jedis.keys("*");
//	System.out.println(set_keys);
	for (Object obj : set_keys)
		{
		String key = obj.toString ();
		String str_json = jedis.get(key);
//		System.out.println (str_json);

		if (str_json != null)
			{
			HashMap <String,String>  unit_aa = json_manipulate.unit_json_parser (str_json);
			System.out.print (key + "\t");
			System.out.print (unit_aa.get ("name") + "\t");
			Object ppx = unit_aa.get ("population");
			System.out.print (ppx + "\t");

			System.out.println (unit_aa.get ("date_mod"));
			}
		}

	System.out.println ("*** 終了 ***");
}
// -------------------------------------------------------------

}
// -------------------------------------------------------------


// -------------------------------------------------------------
